single-line-diagram-force-layout/src/main/java/com/powsybl/sld/force/layout/Vector.java

```diff
@@ -0,0 +1,64 @@
+package com.powsybl.sld.force.layout;
+
+import java.util.Objects;
+
+public class Vector {
```

Comment:
The name of this class is a bit misleading, as vectors is usually something else! but I'd keep it, as in physic we're always speaking of velocity/acceleration/position *vectors*